src/index.js

```diff
@@ -1 +1,20 @@
 import './styles.css';
+import { addScore, getData, createTable } from './score.js';
+
+const add = document.querySelector('.add');
+const user = document.querySelector('.name');
+const score = document.querySelector('.scoreValue');
+const refresh = document.querySelector('.refresh');
+
+add.addEventListener('click', () => {
+  addScore(user.value, score.value);
+});
```

Comment:
- [ ] Since you will be using a `form` to send data to the server, let's get the form element from the DOM and add the `submit` event on it. You will also need to prevent default action. You can learn more about it [here](https://developer.mozilla.org/en-US/docs/Web/API/Event/preventDefault)

Look at this example

```
formElement.addEventListener('submit', (e) => {
  e.preventDefault()
  addScore(user.value, score.value);
});
```